src/handlers/localstorage/ens.ts

```diff
@@ -13,8 +17,57 @@ export const getProfile = async (key: string) => {
 export const saveProfile = (key: string, value: Object) =>
   saveGlobal(ensProfileKey(key), JSON.stringify(value), ensProfileVersion);
 
+export const getProfileImages = async (key: string) => {
+  const images = await getGlobal(
+    ensProfileImagesKey(key),
+    null,
+    ensProfileVersion
+  );
+  return images ? JSON.parse(images) : null;
+};
+
+export const saveProfileImages = (key: string, value: Object) =>
+  saveGlobal(
+    ensProfileImagesKey(key),
+    JSON.stringify(value),
+    ensProfileVersion
+  );
+
+export const getProfileRecords = async (key: string) => {
+  const records = await getGlobal(
+    ensProfileRecordsKey(key),
+    null,
+    ensProfileVersion
+  );
+  return records ? JSON.parse(records) : null;
+};
+
+export const saveProfileRecords = (key: string, value: Object) =>
+  saveGlobal(
+    ensProfileRecordsKey(key),
+    JSON.stringify(value),
+    ensProfileVersion
+  );
+
 export const getResolveName = (key: string) =>
   getGlobal(ensResolveNameKey(key), null);
 
 export const saveResolveName = (key: string, value: string) =>
   saveGlobal(ensResolveNameKey(key), value);
+
+export const getSeenOnchainDataDisclaimer = () =>
+  getGlobal(ensResolveNameKey(ensSeenOnchainDataDisclaimerKey), false);
```

Comment:
does this `ensSeenOnchainDataDisclaimerKey` need to be passed thru `ensResolveNameKey` ? 

same for line 62

Reply:
change it to only use `ensSeenOnchainDataDisclaimerKey`  as `ensProfile.seenOnchainDisclaimer`